Mindo-Joseph/Zen
# #167551831 Setup folder structure

#### What does this PR do?
- This PR contains the folder structure of Zen
#### Description of Task to be completed?
- [x] Setup folder structure
#### How should this be manually tested?
-  Clone the repository
-  Checkout to the ch-setup-folder-structure-167551831
- Observe the folder structure using the tree command
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- [#167551831](https://www.pivotaltracker.com/story/show/167551831)
#### Screenshots (if appropriate)
- N/A
#### Questions:
- None
